### PR TITLE
Remove server stat background jobs

### PR DESCRIPTION
**Pull request by FlorianSW**

### CODE REVIEW

File: rcon/server_stats.py [hunk 1]
```diff
@@ -67,14 +67,6 @@ def save_server_stats_for_last_hours(hours=24, skip_last_hours=1):
     return save_server_stats_for_range(start, end)
 
 
-def save_server_stats_since_inception():
```
Comment (cemathey) [inline rcon/server_stats.py:70]: Why not just leave this one so it doesn't need to get rewritten later? With the CLI disabled and it not being called in `entrypoint.sh` it shouldn't hurt anything.

File: rcon/cli.py [hunk 2]
```diff
@@ -45,11 +45,6 @@ def run_stats_loop():
         sys.exit(1)
 
 
-@cli.command(name="record_server_stats_inception")
```
Comment (cemathey) [inline rcon/cli.py:48]: Maybe just comment this out? This stuff is getting re-enabled at some point I assume.

File: entrypoint.sh [hunk 3]
```diff
@@ -32,8 +32,6 @@ then
     exit 0
 fi
   sleep 10
-  LOGGING_FILENAME=historical_server_stats_$SERVER_NUMBER.log python -m rcon.cli record_server_stats_inception &
```
Comment (cemathey) [inline entrypoint.sh:35]: Maybe just comment this out? This stuff is getting re-enabled at some point I assume.

Comment (MarechJ) [reply entrypoint.sh:35]: it's fine to remove this. However why remove the code also?

**Review by cemathey — APPROVED** [review]

**Pull request merged**